### ISSUE TITLE
Update dotsettings

### ISSUE THREAD

**Issue by markgould** [JoshClose/CsvHelper#800]
I noticed that your formatting style is tabs and mine is spaces. I verified I'm using your dotsettings file but the tabs/indents says to use the style/size from VS. Unless I'm doing something wrong, you might want to change that to your settings so when others contribute they will use your settings. 

![image](https://user-images.githubusercontent.com/2712328/31789971-7662f4a6-b4d9-11e7-9403-99febe4019a3.png)

**Comment by JoshClose** [comment]
I actually don't use Resharper anymore. I should just supply a visual studio file for people to use.

**Comment by jthelin** [comment]
FYI, here is the `.vssettings` file from the NET Core (corefx) project, which now seems to be widely used by many / most of the DotNet Foundation projects and others on GitHub.

https://github.com/dotnet/corefx/blob/master/corefx.vssettings

Personally, I have found it to provides some "reasonable" baseline settings for C# OSS projects.

**Comment by JoshClose** [comment]
Agreed, but I generally like a little more white space around things. I find it easier to read. 

I also like tabs because every user can then set how much space they want the tab to take up. I like 4 spaces, but some people like 2. 2 isn't enough for me to easily differentiate. This is in visual studio though. A tab on github is huge, unfortunately, but there are ways around that.

**Comment by jthelin** [comment]
I'm not getting into the tabs-vs-spaces discussion 😫 at all, and at the end of the day it is your call Josh what to use on your project.

The suggestion I was making was to grab a well known "baseline" file from corefx and then just customize the particular settings you want for this project.

For example:
``` xml
<PropertyValue name="TabSize">4</PropertyValue>
<PropertyValue name="InsertTabs">true</PropertyValue>
<PropertyValue name="IndentSize">4</PropertyValue>
```
[and may be some other settings you want too...]

The Visual Studio 2017 menu item for reading project code style settings is 
Tools -> Import and Export Settings... -> Import -> No [or Yes] -> Browse... 
and then VS will start showing warning about code style variations.

**Comment by JoshClose** [comment]
Thanks.

I would like to hear your argument for spaces though, if you have one. I haven't heard one that has made me switch yet, but I'm not opposed to it. Setting up my changes all the time is a little annoying... to the point where i'm considering just switching to all defaults and using that.

**Comment by jamesbascle** [comment]
Spaces is my preference as well, but there's no reason to change it now.
We'll just lose all context and history.
